### PR TITLE
Store language in app state

### PR DESCRIPTION
**Pull request by eatyourgreens**
Language selector stores language in app state.

Language is always an object of form `{label, value}`, never a string.

### CODE REVIEW

File: src/containers/ProjectDashboardContainer.jsx [hunk 1]
```diff
@@ -45,32 +41,34 @@ class ProjectDashboardContainer extends Component {
...
   onChangeLanguage(option) {
-    this.setState({ option });
+    const { actions } = this.props;
+    actions.setLanguage(option);
   }
 
   render() {
     const project = this.props.project.data;
-    const { fieldguides, pages, workflows, tutorials } = this.props.project;
+    const { fieldguides, language, pages, workflows, tutorials } = this.props.project;
```
Comment (simoneduca) [inline src/containers/ProjectDashboardContainer.jsx]: `language` should be added in `propTypes`

**Comment by eatyourgreens** [comment]
Sure. I was following the file structure you'd set up originally, but would prefer `propTypes` and `defaultProps` defined after the component definition, if that's ok?

**Comment by simoneduca** [comment]
Sounds good.

**Comment by eatyourgreens** [comment]
Cool. I'll do that in a separate PR, for everything.

**Comment by simoneduca** [comment]
Same here, leaving the branch in case you want to add `language` to `propTypes`.

**Comment by eatyourgreens** [comment]
I thought 2064259 fixed that? Did I miss one?

**Comment by simoneduca** [comment]
Actually, you're right. I thought GH didn't show comments anymore once the files get updated, but the comment is still there. Bah.

**Comment by eatyourgreens** [comment]
I think that only happens if that particular section of code is changed by a subsequent commit. It would probably have been hidden if you'd commented on the `propTypes` object.

**Comment by simoneduca** [comment]
Makes more sense.

**Pull request merged**